### ISSUE TITLE
Video playback broken under Linux (blank)

### ISSUE THREAD

**Issue by sofian** [mapmapteam/mapmap#124]
Under Linux, the latest checkout of develop results in the video not playing back well. I did a git bisect to find out that the error comes from changes introduced as part of the following commit: d1f63b8501cd7ee2f4d28ce91edfd246985ebcb8

**Comment by sofian** [comment]
tats@antilope:~/Documents/workspace/mapmap$ git bisect bad
d1f63b8501cd7ee2f4d28ce91edfd246985ebcb8 is the first bad commit
commit d1f63b8501cd7ee2f4d28ce91edfd246985ebcb8
Author: Vasilis Liaskovitis <vliaskov@gmail.com>
Date:   Mon Feb 9 21:44:47 2015 +0100

    Fix issue #114: Audio playback of the video files played
    
    We now create an audio pipeline with a volume gstreamer element. The "volume"
    property of the element has a range of 0.0 - 1.0 and is 0.0 by default (i.e.
    effectively muted input). The property can be seen and changed in the paint
    property browser (in %)

:100644 100644 84bb901ba5413f38a8a8df8ba4721d05772c10bc f401493df22f33aa3ad658849c165fa77e345003 M	MediaImpl.cpp
:100644 100644 49a666c95a0d4c1a4b225b82be535a1636bbf230 d7c8a6bffe17764664a00348c566094741f1997b M	MediaImpl.h
:100644 100644 2b7609935a0fe5642c4b3925616b2f4dd032e779 a4c954cf2d10bba4611a40b39c31e19e126e6a87 M	Paint.cpp
:100644 100644 19b69f65bed9db604b759f6200d0fee294937219 0a4e33751ff7f36cf01c6fd1f44d8de176ff2004 M	Paint.h
:100644 100644 9e4059c7cb9c268c349f4cedb5d45bd88ebc31fd ab9b2797c217f3e866cfd71c2d59adb829e533b6 M	PaintGui.cpp
:100644 100644 2b8a729018634bee8d05738c4adb955e2bae1545 ff6c08ea8de60883d469fb315f9203c0005b9690 M	PaintGui.h

**Comment by sofian** [comment]
The faulty line seems to be MediaImpl.cpp line 453:

```
  // Build the pipeline. Note that we are NOT linking the source at this
  // point. We will do it later.
  gst_bin_add_many (GST_BIN (_pipeline),
    _isSharedMemorySource ? _shmsrc0 : _uridecodebin0, _queue0,
    _videoconvert0, videoscale0, capsfilter0, _appsink0,
    _audioqueue0, _audioconvert0, _audioresample0, _audiovolume0, _audiosink0, /// <---- this line
    NULL);
```

**Comment by aalex** [comment]
I believe I fixed it yesterday.

**Comment by aalex** [comment]
Probably fixed in 0.3.1
